lua/lspconfig/configs/ltex_plus.lua

```diff
@@ -0,0 +1,93 @@
+local util = require 'lspconfig.util'
+
+local language_id_mapping = {
+  bib = 'bibtex',
+  pandoc = 'markdown',
+  plaintex = 'tex',
+  rnoweb = 'rsweave',
+  rst = 'restructuredtext',
+  tex = 'latex',
+  text = 'plaintext',
+}
+
+local function get_language_id(_, filetype)
+  return language_id_mapping[filetype] or filetype
+end
+
+return {
+  default_config = {
+    cmd = { 'ltex-ls-plus' },
+    filetypes = {
+      'bib',
+      'context',
+      'gitcommit',
+      'html',
+      'markdown',
+      'org',
+      'pandoc',
+      'plaintex',
+      'quarto',
+      'mail',
+      'mdx',
+      'rmd',
+      'rnoweb',
+      'rst',
+      'tex',
+      'text',
+      'typst',
+      'xhtml',
+    },
+    root_dir = util.find_git_ancestor,
+    single_file_support = true,
+    get_language_id = get_language_id,
+    settings = {
+      ltex = {
+        enabled = {
+          'bib',
+          'context',
+          'gitcommit',
+          'html',
+          'markdown',
+          'org',
+          'pandoc',
+          'plaintex',
+          'quarto',
+          'mail',
+          'mdx',
+          'rmd',
+          'rnoweb',
+          'rst',
+          'tex',
+          'text',
+          'typst',
+          'xhtml',
+        },
+      },
+    },
+  },
+  docs = {
+    description = [=[
+https://github.com/ltex-plus/ltex-ls-plus
+
+LTeX Language Server: LSP language server for LanguageTool 🔍✔️ with support for LaTeX 🎓, Markdown 📝, and others
+
+To install, download the latest [release](https://github.com/ltex-plus/ltex-ls-plus) and ensure `ltex-ls-plus` is on your path.
+
+This server accepts configuration via the `settings` key.
+
+```lua
+  settings = {
+		ltex = {
+			language = "en-GB",
+		},
+	},
```

Comment:
please follow the codebase conventions. take a hint from the other code in this repo.
```suggestion
  settings = {
    ltex = {
      language = "en-GB",
    },
  },
```

Reply:
Right, sorry!